docs/reference.md

```diff
@@ -430,6 +433,21 @@ OpenWhisk has a few system limits, including how much memory an action uses and
 * A user cannot change this limit when creating the action.
 * A CLI call that exceeds this limit receives an error code corresponding to TOO_MANY_REQUESTS.
 
+### Size of the code per action (Fixed: 48MB)
```

Comment:
I think this means that you need to remove the document the limitation that we already have about action artifacts cant' be larger than 1MB, if I understand this limitation correctly

today we have this in the doc both table and text:
Per action artifact (MB) (Fixed: 1MB)

The maximum code size for the action is 1MB.
It is recommended for a JavaScript action to use a tool to concatenate all source code including dependencies into a single bundled file.